stock_putaway_product_form/__init__.py

```diff
@@ -0,0 +1,4 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import models
+from . import tests
```

Comment:
You do not need import tests

Reply:
In 12.0 these line exists, IMO isn't correct and need to remove, I will have to create new PR about it, right?

Reply:
Ok @victoralmau